sdk/spring/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/repository/support/StringBasedCosmosQuery.java

```diff
@@ -91,7 +91,8 @@ public Object execute(final Object[] parameters) {
             }
         }
 
-        SqlQuerySpec querySpec = new SqlQuerySpec(expandedQuery, sqlParameters);
+        String formattedQuery = expandedQuery.replaceAll("\\s+{1,}", " ");
```

Comment:
It would be good to have some unit tests covering this regex. Can you please add those?

Reply:
Done